Bindings/Java/tests/TestBasics.java

```diff
@@ -23,6 +23,12 @@ public static void main(String[] args) {
 
     m.addComponent(pj);
 
+    // Change from ArrayPtrs<Object*> being non-const const.
```

Comment:
Yes I meant to leave it in, since this is similar to the code in the GUI that I wanted to make sure would work. The comment describing this "test" is lacking, though.